__tests__/utils/resizingConstraints.test.js

```diff
@@ -0,0 +1,35 @@
+const { calculateResizingConstraintValues, resizingConstraintValues } = require('../../utils/resizingConstraints.js');
+
+describe('Test sizing constraints', () => {
+  test('reduce series of numbers to their bitwise AND value', () => {
+    const { top, left, height } = resizingConstraintValues;
+    const input = [left, height];
+    // eslint-disable-next-line no-bitwise
+    const output = top & left & height;
+
+    expect(calculateResizingConstraintValues(top, ...input)).toEqual(output);
+  });
+
+  test('given undefined values will return 0', () => {
```

Comment:
Should this throw? Is 0 a valid resizingConstraint value?

Reply:
AND bitwise operator with undefined returns 0, but you are right and if the reduce returns 0 I use `none` as default resizing constraint value.